jee-commons/src/main/java/biz/paluch/jee/commons/JndiLookupStrategy.java

```diff
@@ -16,8 +16,7 @@
     public <T> T doLookup(String name) {
         try {
             return InitialContext.doLookup(name);
-        } catch (NamingException e) {
-            throw new IllegalArgumentException(e);
-        }
+        } catch (NamingException e) {}
```

Comment:
Yes, but solving it by throwing IllegalStateException as suggested below would be an incompatible API change, too.